jekyll/_cci2/monitoring.adoc

```diff
@@ -30,6 +34,8 @@ Following are the enabled metrics:
 
 https://www.nomadproject.io/docs/telemetry/metrics.html#job-metrics[Nomad job metrics] are enabled and emitted by the Nomad Server agent. Five types of metrics are reported:
 
+// Do customers get these metics in Cloudwatch/Datadog too?
```

Comment:
They can but not by default.

Reply:
Should we tell them how? How do they get the metrics by default - not at all?

Reply:
Sorry that question makes no sense at all!
I mean should we tell them how to 'get' these metrics if they don't go to datadog/cloudwatch by default?